cmd/sabayon/main.go

```diff
@@ -21,6 +21,10 @@ func main() {
 	flag.Parse()
 
 	var domains = strings.Split(os.Getenv("ACME_DOMAIN"), ",")
+	// trim potential whitespace
+	for _, domain := range domains {
+		domain = strings.TrimSpace(domain)
```

Comment:
this will not work. `domain` is a local variable. So it won't update the array.
You need to do actually update the entry:

```
for i, domain := range domains {
  domains[i] = strings.TrimSpace(domain)
}
```